packages/wdio-config/tests/utils.test.ts

```diff
@@ -64,4 +64,10 @@ describe('utils', () => {
             expect(isCloudCapability({})).toBe(false)
         })
     })
-})
+
+    describe('validateTsConfigPaths', () => {
+        it('should throw error if valid Tsconfig not provided', ()  => {
+            expect(() => validateTsConfigPaths({ project: './test/tsconfig.json' })).toThrow('Provided tsconfig file path in wdio config is incorrect. Is it correctly set in wdio config ?')
```

Comment:
Let's shorten this a bit.
```suggestion
            expect(() => validateTsConfigPaths({ project: './test/tsconfig.json' }))
                .toThrow(/Is it correctly set in wdio config/)
```